content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -7,200 +7,244 @@ section: doc
+
+NOTE: If you already have a Kubernetes cluster, you can skip to <<Deploying the IBM Cloud Block Storage plug-in>>.
+
+To provision a Kubernetes cluster:
+
+. Select *Catalog* in the upper left corner.
+. Select *Service* from the catalog.
+. Search for *Kubernetes Service* and select it.
++
+image::tutorials/IBMCloud/kubernetes-select.png[IBM Kubernetes selection]
++
+You are now at the Kubernetes deployment page, and you need to specify some details about the cluster.
+
+. Choose a plan type of *standard* or *free*.
+The free plan only has one worker node and no subnet.
+. To provision a standard cluster, you will need to upgrade your account to Pay-As-You-Go.
+To upgrade to a Pay-As-You-Go account, complete the following steps:
+.. In the console, go to *Manage > Account*
+.. Select *Account settings*, and then select *Add credit card*.
+.. Enter your payment information, select *Next*, and then submit your information.
+. Choose *Classic* or *VPC*, read the link:https://cloud.ibm.com/docs/containers?topic=containers-infrastructure_providers[docs], and then choose the most suitable type for yourself.
++
+image::tutorials/IBMCloud/infra-select.png[IBM infrastructure selection]
+
+. Now choose your location settings.
+For more information, please refer to link:https://cloud.ibm.com/docs/containers?topic=containers-regions-and-zones#zones[Locations].
+.. Choose *Geography* (continent).
++
+image::tutorials/IBMCloud/location-geo.png[IBM geo location,]
+
+.. Choose *Single* or *Multizone*.
+In single zone, your data is kept in only one datacenter.
+With Multizone, it is distributed to multiple datacenters, and thus safer in an unforeseen zone failure.
++
+image::tutorials/IBMCloud/location-avail.png[IBM available locations]
+
+.. Choose a *Worker Zone* if using Single zones or **Metro** if using Multizone.
++
+image::tutorials/IBMCloud/location-worker.png[IBM worker location]
+
+.. If you wish to use Multizone, set up your account with link:https://cloud.ibm.com/docs/dl?topic=dl-overview-of-virtual-routing-and-forwarding-vrf-on-ibm-cloud[VRF] or link:https://cloud.ibm.com/docs/vlans?topic=vlans-vlan-spanning#vlan-spanning[enable VLAN spanning].
++
+NOTE: If at your current location selection, there is no available Virtual LAN, a new VLAN is created for you.
+
+. Choose a *Worker node setup* or use the pre-selected one, and then set *Worker node amount per zone*.
++
+image::tutorials/IBMCloud/worker-pool.png[IBM worker-pool]
+
+. Choose *Master Service Endpoint*.
+
+.. Choose public-only to make your controller publicly accessible. 
+.. In VRF-enabled accounts, you can choose private-only to make your controller accessible on the private network or via VPN tunnel.
+When you have a VRF-enabled account, your cluster is set up by default to use both private and public endpoints.
+For more information, refer to link:https://cloud.ibm.com/docs/account?topic=account-service-endpoints-overview[endpoints].
```

Comment:
Now that I read this more closely, I wonder if we should restructure this slightly?

```suggestion
. Choose a *Master service endpoint*.
+
NOTE: When you have a VRF-enabled account, your cluster is set up to use both private and public endpoints by default. 
You can select **Private endpoint only** to make your controller accessible on the private network or via VPN tunnel. 
You can select **Public endpoint only** to make your controller publicly accessible. 
For more information, refer to link:https://cloud.ibm.com/docs/account?topic=account-service-endpoints-overview[endpoints].
```